packages/frontend/src/redux/createReducers/combinedAccountReducers.js

```diff
@@ -0,0 +1,5 @@
+import tokensSlice from '../slices/tokens';
```

Comment:
My idea was to keep the whole functionality regarding keying by `accountId` as a separate layer of functionality, so while developing you don't have to worry about it or know how it works. 

For example, if someone would like to add a new reducer, he just opens `combinedAccountReducers.js` or `combinedMainReducers.js` (like before) and he doesn't have to worry about `setupAccountReducer` at all. If we add it inline `setupAccountReducer` function it will be much less intuitive.

So It's a separate module because the code will be organized better.